util/src/main/java/org/killbill/billing/util/dao/TableName.java

```diff
@@ -35,6 +35,7 @@
     INVOICE_ITEMS("invoice_items", ObjectType.INVOICE_ITEM),
     INVOICE_PAYMENTS("invoice_payments", ObjectType.INVOICE_PAYMENT),
     INVOICES("invoices", ObjectType.INVOICE),
+    INVOICE_PARENT_CHILDREN("invoice_parent_children", ObjectType.INVOICE_PARENT_CHILD),
```

Comment:
`ObjectType.INVOICE_PARENT_CHILD` should not be exposed at the API level, since the table entries won't be exposed directly to the APIs.